src/EnergyPlus/HVACHXAssistedCoolingCoil.cc

```diff
@@ -636,86 +531,78 @@ namespace HVACHXAssistedCoolingCoil {
                 }
             }
 
-            TestCompSet(state,
-                        state.dataHVACAssistedCC->HXAssistedCoil(HXAssistedCoilNum).HXAssistedCoilType,
-                        state.dataHVACAssistedCC->HXAssistedCoil(HXAssistedCoilNum).Name,
-                        state.dataLoopNodes->NodeID(SupplyAirInletNode),
-                        state.dataLoopNodes->NodeID(SecondaryAirOutletNode),
-                        "Air Nodes");
-
-            state.dataHVACAssistedCC->HXAssistedCoil(HXAssistedCoilNum).HXAssistedCoilInletNodeNum =
-                GetOnlySingleNode(state,
-                                  state.dataLoopNodes->NodeID(SupplyAirInletNode),
-                                  ErrorsFound,
-                                  DataLoopNode::ConnectionObjectType::CoilSystemCoolingDXHeatExchangerAssisted,
-                                  state.dataHVACAssistedCC->HXAssistedCoil(HXAssistedCoilNum).Name,
-                                  DataLoopNode::NodeFluidType::Air,
-                                  DataLoopNode::ConnectionType::Inlet,
-                                  NodeInputManager::CompFluidStream::Primary,
-                                  ObjectIsParent);
-            CoolingCoilInletNodeNum = GetOnlySingleNode(state,
-                                                        state.dataLoopNodes->NodeID(SupplyAirOutletNode),
-                                                        ErrorsFound,
-                                                        DataLoopNode::ConnectionObjectType::CoilSystemCoolingDXHeatExchangerAssisted,
-                                                        state.dataHVACAssistedCC->HXAssistedCoil(HXAssistedCoilNum).Name,
-                                                        DataLoopNode::NodeFluidType::Air,
-                                                        DataLoopNode::ConnectionType::Internal,
-                                                        NodeInputManager::CompFluidStream::Primary,
-                                                        ObjectIsParent);
```

Comment:
[src/EnergyPlus/HVACHXAssistedCoolingCoil.cc:656]:(style),[unreadVariable],Variable 'CoolingCoilInletNodeNum' is assigned a value that is never used.